### DISCUSSION TITLE
Zstd compression support

### DISCUSSION THREAD

**Discussion by AlmogAlmado** [redpanda-data/connect#2322] [Ideas]
Is there any intention to adopt Zstd compression in the future? 
It has excellent performance and widespread adoption in the industry.

**Comment by mihaitodor** [comment]
Hey @AlmogAlmado, it's available in Bloblang via [`compress()`](https://www.benthos.dev/docs/guides/bloblang/methods#compress) and [`decompress()`](https://www.benthos.dev/docs/guides/bloblang/methods#decompress) (the algorithm is labelled `zstd`). Currently, it's a bit hard to discover and Algolia doesn't seem to index that documentation page currently for some reason. We'll look into getting this sorted.

**Reply by AlmogAlmado** [reply]
Thanks, @mihaitodor! appreciate the quick assistance.
